tests/Macros/PaginateTest.php

```diff
@@ -1,11 +1,10 @@
 <?php
 
-namespace Spatie\CollectionMacros\Test\Macros;
+namespace Spatie\CollectionMacros\Test;
```

Comment:
Since the test-case is in the `Macros`-folder, the namespace should reflect that:
```suggestion
namespace Spatie\CollectionMacros\Test\Macros;
```